packages/auth/src/core/strategies/strategy.test.ts

```diff
@@ -17,17 +20,20 @@ describe("redirection", () => {
     },
     saml: {
       builder: () => new SAMLStrategy(),
-      uri: buildMockedRedirectionURL("saml"),
+      uri: buildMockedRedirectionURL("saml") + "&state=/redirect-path",
     },
   } as const;
 
-  it.each(Object.keys(cases))("initializes authentication (%s)", (k) => {
+  it.each(Object.keys(cases))("initializes authentication (%s)", async (k) => {
     const key = k as keyof typeof cases;
-    const strategy = cases[key].builder();
-    const result = strategy.authenticate(mockAuthConfig, {
+    const strategy = cases[key].builder() as AbstractStrategy;
+    const result = await strategy.authenticate(mockAuthConfig, {
```

Comment:
ここのasとawaitはコードを見ると不要に見えました。

```suggestion
    const strategy = cases[key].builder();
    const result = strategy.authenticate(mockAuthConfig, {
```